pkg/apps/alert/latest/versions.go

```diff
@@ -0,0 +1,50 @@
+/*
+Copyright (C) 2019 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package alert
+
+import "fmt"
+
+// imageTags is a map of the Alert versions to it's images and tags
+var imageTags = map[string]map[string]string{
+	"3.1.0": {
+		"blackduck-alert": "3.1.0",
+		"blackduck-cfssl": "1.0.0",
+	},
+	"4.0.0": {
+		"blackduck-alert": "4.0.0",
+		"blackduck-cfssl": "1.0.0",
+	},
+}
+
+// getImageTag returns the url for an image
+func (c *Creater) getImageTag(version, name string) string {
```

Comment:
OK but you don't actually use this function to get the container tags. 
The CRD should probably be modified to have a version field.

Reply:
I removed the imageTag function and added a Version field to Alert